locales/de_CH/LC_MESSAGES/duckduckgo.po

```diff
@@ -5073,5 +5069,9 @@ msgid "Anonymous location enabled."
 msgstr "Anonymer Standort aktiviert."
 
 msgctxt "precise_user_location"
-msgid "Location information is stored only on your device. %sMore info%s"
-msgstr "Standortinformationen werden nur auf deinem Gerät gespeichert. %sWeitere Informationen%s"
+msgid "Location information is stored only on your device."
+msgstr "Standortinformationen werden nur auf deinem Gerät gespeichert."
+
+msgctxt "precise_user_location"
+msgid "More info"
+msgstr "Weitere Informationen"
```

Comment:
@gtfiorentino From the PR in dub I understand that you removed this `precise_user_location` More info to rely on the one that is already existing. If that is the case, you should probably remove them from here as well (and from all other files in this PR).

Reply:
The MORE_INFO token is still needed. The SHOW_MORE token is the one that was removed.